output.c

```diff
@@ -49,15 +49,15 @@ void print_test_footer(
     }
 }
 
-void open_output_file(char* file_name) {
+void open_output_file(char* output_file_path) {
```

Comment:
Include `output_` to the argument isn't needed when the function name is `open_output_file`. Plus there's only one file path input to the function. However, `file_path` is better than `file_name`.

Reply:
I changed `output_file_path` to `file_path` and also `config_file_path` to `file_path` in `load_conf_file` or else the name of the argument of the two functions would become inconsistent (which is also why I did this change in the first place).